nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/registry/context/RenderingContextImpl.java

```diff
@@ -331,6 +333,12 @@ public void addParameterListValues(String name, List<?> values) {
         parameters.computeIfAbsent(realName, key -> new CopyOnWriteArrayList<>()).addAll(values);
     }
 
+    @Override
+    public APIVersion getAPIVersion() {
+        APIVersion apiVersion = getParameter(API_VERSION_ATTRIBUTE_NAME);
+        return apiVersion != null ? apiVersion : APIVersion.latest();
```

Comment:
What do you think about:
```suggestion
        return Objects.requireNonNullElseGet(apiVersion, APIVersion::latest);
```